docs/_sources/index.rst.txt

```diff
@@ -62,22 +64,30 @@ MHKiT is developed as a collaboration between the National Renewable Energy Labo
 
 Funding
 ================
-Development of MHKiT is funded by the U.S. Department of Energy's Water Power Technologies Office. 
-The copyright of MHKiT is held by the National Renewable Energy Laboratory, 
+<<<<<<< Updated upstream
+The development of MHKiT is funded by the U.S. Department of Energy's Water Power Technologies Office.
+=======
+Development of MHKiT is funded by the U.S. Department of Energy's Water Power Technologies Office.
+>>>>>>> Stashed changes
+The copyright of MHKiT is held by the National Renewable Energy Laboratory,
 Pacific Northwest National Laboratory, and Sandia National Laboratories.
 The software is distributed under the :ref:`License:Revised BSD License`.
 
-The `National Renewable Energy Laboratory <https://www.nrel.gov/>`_ is a national laboratory of the U.S. Department of Energy, Office of Energy Efficiency and Renewable Energy, operated by the Alliance for Sustainable Energy, LLC. under contract No. DE-AC36-08GO28308. 
+The `National Renewable Energy Laboratory <https://www.nrel.gov/>`_ is a national laboratory of the U.S. Department of Energy, Office of Energy Efficiency and Renewable Energy, operated by the Alliance for Sustainable Energy, LLC. under contract No. DE-AC36-08GO28308.
 
 `Pacific Northwest National Laboratory <https://www.pnnl.gov/>`_ is operated by Battelle for the United States Department of Energy under Contract DE-AC05-76RL01830.
 
-`Sandia National Laboratories <http://www.sandia.gov/>`_ is a multi-mission laboratory managed and operated by National Technology 
-and Engineering Solutions of Sandia, LLC., a wholly owned subsidiary of Honeywell International, Inc., for the U.S. Department of Energy's National 
+`Sandia National Laboratories <http://www.sandia.gov/>`_ is a multi-mission laboratory managed and operated by National Technology
+<<<<<<< Updated upstream
+and Engineering Solutions of Sandia, LLC., a wholly-owned subsidiary of Honeywell International, Inc., for the U.S. Department of Energy's National
+=======
+and Engineering Solutions of Sandia, LLC., a wholly owned subsidiary of Honeywell International, Inc., for the U.S. Department of Energy's National
+>>>>>>> Stashed changes
```

Comment:
@SeanPluemer  Looks like you made some errors here.  Please fix it here and anywhere else.
![image](https://user-images.githubusercontent.com/13438942/81337321-0ef29980-9068-11ea-98ef-05ea21a21a6f.png)